includes/init_includes/init_customer_auth.php

```diff
@@ -79,27 +79,34 @@
  * 2 = customer authorization pending can browse but no prices
  */
 switch (true) {
+  /**
+   * bypass redirects for these scripts, to processing regardless of store mode or cust auth mode
+   */
+  case (preg_match('/ipn_main_handler\.php$/', $_SERVER['SCRIPT_NAME'])):

```

Comment:
The intent was that it would be directory-agnostic, thus leaving off the ^ in the regex pattern.

Is there a use-case where this fails to work as intended?